apps/tutorial-servo-can-control/main.cpp

```diff
@@ -0,0 +1,26 @@
+#include "TutorialServo.h"
+#include "CANBus.h"
+#include "CANMsg.h"
+#include "mbed.h"
+
+int main()
+{
+    CANBus can(CAN1_RX, CAN1_TX, HWBRIDGE::ROVERCONFIG::ROVER_CANBUS_FREQUENCY);
+    CANMsg rxMsg;
+    float rangeOfMotion = NULL;
+
+    //listens on can for any incoming can messages and if any places into rxMsg object
+    while(true)
+    {
+        if(can.read(rxMsg))
+        {
+            rxMsg.getPayload(rangeOfMotion);
+        }
+    }
+
+    TutorialServo servoObject;
```

Comment:
be careful because these lines of code will never be run (your code will enter the while loop and never leave)